themes/digital.gov/src/scss/new/_content.scss

```diff
@@ -42,11 +42,11 @@
     clear: both;
   }
 
-  // Code
-  code {
-    @include u-padding-x("05");
-    @include u-padding-y("1px");
-    @include u-bg("base-lightest");
+  // Inline code
+  // Example: `/resources/guide-create-mobile-friendly-websites`
+  p > code {
+    @include u-padding-x(0.5);
+    @include u-bg("white");
     @include u-font("mono", "xs");
     @include u-line-height("mono", 1);
     @include u-border("1px", "solid", "base-lighter");
```

Comment:
There is a `<script>` used in a `ul li` that doesn't render. We should make this selector more general like `code` instead of `p > code` to target those.

<img width="717" alt="Screenshot 2024-04-25 at 9 31 34 AM" src="https://github.com/GSA/digitalgov.gov/assets/104778659/9d06ac33-ab6c-45da-aba0-8c84ed4b5e6a">

Reply:
@nick-mon1 a more general selector causes issues to the block. Captured in this [discussion](https://github.com/GSA/digitalgov.gov/pull/7519#discussion_r1576452205).

If we can find a new selector that doesn't cause issues in _either_ scenario would be good. Ideally, the inline code blocks would have their own class.

Reply:
updated to capture the ul elements